nuxeo-ai-model/src/test/java/org/nuxeo/ai/model/TestDatasetOperation.java

```diff
@@ -146,8 +147,16 @@ public void shouldRunRestartOperation() throws OperationException {
 
         int split = 60;
         params.put("query", TEST_QUERY);
-        params.put("inputs", "dc:title,dc:description");
-        params.put("outputs", "dc:nature");
+
+        Properties inputs = new Properties();
+        inputs.put("dc:title", "txt");
+        inputs.put("dc:description", "txt");
+        params.put("inputProperties", inputs);
+
+        Properties outputs = new Properties();
+        outputs.put("dc:nature", "cat");
+        params.put("outputProperties", outputs);
+
```

Comment:
Have a property that will be a blob.

Reply:
blob processing covered in a different place, it's simple rewrite of deprecated properties